cms/static/sass/views/_settings.scss

```diff
@@ -811,7 +811,7 @@
         .field {
           display: inline-block;
           vertical-align: top;
-          width: flex-grid(3, 6);
+          width: flex-grid(2, 6);
```

Comment:
> that was in your npoed PR

As I mentioned before, there is a bit different UI and it looks like you don't need it here.